### PR TITLE
Add operations introduced in Protocol 11.

### LINKED ISSUE #454: Update list of operations for Protocol v11
Add documentation for `ManageBuyOffer` and `ManageSellOffer`. Update links in Stellar lab to the [list of operations page](https://www.stellar.org/developers/guides/concepts/list-of-operations.html#manage-offer) to reference the new docs. [This page](https://www.stellar.org/developers/horizon/reference/resources/operation.html) has already been updated, it might be a good reference.

### PR DESCRIPTION
**Pull request by theaeolianmachine**
* Change ManageOffer -> ManageSellOffer
* Change CreatePassiveOffer -> CreatePassiveSellOffer
* Add ManageBuyOffer

Fixes #454.

### CODE REVIEW

File: guides/concepts/list-of-operations.md [hunk 1]
```diff
@@ -131,75 +133,122 @@ If you want to delete an existing offer set Offer ID to existing offer ID and se
...
+| MANAGE_BUY_OFFER_BUY_NOT_AUTHORIZED | -4 | The account creating the offer is not authorized to sell this asset. |
+| MANAGE_BUY_OFFER_BUY_NOT_AUTHORIZED | -5 | The account creating the offer is not authorized to buy this asset. |
+| MANAGE_BUY_OFFER_LINE_FULL | -6 | The account creating the offer does not have sufficient limits to receive `buying` and still satisfy its buying liabilities. |
+| MANAGE_BUY_OFFER_UNDERFUNDED | -7 | The account creating the offer does not have sufficient limits to send `selling` and still satisfy its selling liabilities. Note that if selling XLM then the account must additionally maintain its minimum XLM reserve, which is calculated assuming this offer will not completely execute immediately. |
+| MANAGE_BUY_OFFER_CROSS_SELF | -8 | The account has opposite offer of equal or lesser price active, so the account creating this offer would immediately cross itself. |
+| MANAGE_BUY_OFFER_BUY_NO_ISSUER | -9 | The issuer of selling asset does not exist. |
+| MANAGE_BUY_OFFER_BUY_NO_ISSUER | -10 | The issuer of buying asset does not exist. |
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: These two error codes duplicated

File: guides/concepts/list-of-operations.md [hunk 2]
```diff
@@ -131,75 +133,122 @@ If you want to delete an existing offer set Offer ID to existing offer ID and se
...
-Passive offers allow market makers to have zero spread. If you want to trade EUR for USD at 1:1 price and USD for EUR also
- at 1:1, you can create two passive offers so the two offers don't immediately act on each other.
-
-Once the passive offer is created, you can manage it like any other offer using the [manage offer](#manage-offer) operation.
+| ----- | ---- | ------ |
+| MANAGE_BUY_OFFER_MALFORMED | -1 | The input is incorrect and would result in an invalid offer. |
+| MANAGE_BUY_OFFER_BUY_NO_TRUST | -2 | The account creating the offer does not have a trustline for the asset it is selling. |
+| MANAGE_BUY_OFFER_BUY_NO_TRUST | -3 | The account creating the offer does not have a trustline for the asset it is buying. |
+| MANAGE_BUY_OFFER_BUY_NOT_AUTHORIZED | -4 | The account creating the offer is not authorized to sell this asset. |
+| MANAGE_BUY_OFFER_BUY_NOT_AUTHORIZED | -5 | The account creating the offer is not authorized to buy this asset. |
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: These two error codes duplicated

File: guides/concepts/list-of-operations.md [hunk 3]
```diff
@@ -131,75 +133,122 @@ If you want to delete an existing offer set Offer ID to existing offer ID and se
...
-offer is of the same price as your passive offer.
-
-Passive offers allow market makers to have zero spread. If you want to trade EUR for USD at 1:1 price and USD for EUR also
- at 1:1, you can create two passive offers so the two offers don't immediately act on each other.
-
-Once the passive offer is created, you can manage it like any other offer using the [manage offer](#manage-offer) operation.
+| ----- | ---- | ------ |
+| MANAGE_BUY_OFFER_MALFORMED | -1 | The input is incorrect and would result in an invalid offer. |
+| MANAGE_BUY_OFFER_BUY_NO_TRUST | -2 | The account creating the offer does not have a trustline for the asset it is selling. |
+| MANAGE_BUY_OFFER_BUY_NO_TRUST | -3 | The account creating the offer does not have a trustline for the asset it is buying. |
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: These two error codes duplicated. I recommend comparing against [the XDR file](https://github.com/stellar/stellar-core/blob/df28f6481c211f0812cb2e5e2132df0d461f13a3/src/xdr/Stellar-transaction.x#L558) for correctness. Here you want
```
 MANAGE_BUY_OFFER_SELL_NO_TRUST = -2, // no trust line for what we're selling
MANAGE_BUY_OFFER_BUY_NO_TRUST = -3, // no trust line for what we're buying
```

**Comment by theaeolianmachine** [comment]
Yeah will do — it's not my editor, it's actually me. Most of the docs were written without knowing what a linebreak is, and they're impossible to read in a text editor. I tried not reformatting too much, but yeah some still made it in there. Probably makes sense to do an entire reformat pass of all the docs at some point.

File: guides/concepts/list-of-operations.md (hunk 3 above)
Comment (theaeolianmachine) [reply guides/concepts/list-of-operations.md]: Yeah this came about via a find and replace, good catch.

**Pull request merged**